src/PortalModal.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react';
+import ReactDOM from 'react-dom';
+
+interface PortalModalProps {
+    children: ReactNode,
+};
+
+const Modal = ({ 
+    children 
+}: PortalModalProps) => {
```

Comment:
React에는 PropsWithChildren이라는 type을 제공해요. 저 타입에는 children prop이 있어서 별도로 interface를 설정할 필요가 사라져요.
```suggestion
const Modal = ({ 
    children 
}: React.PropsWithChildren) => {
```